### DISCUSSION TITLE
Only Getting Mono Audio over HDMI

### DISCUSSION THREAD

**Discussion by Aaron407s** [opencardev/crankshaft#460] [Q&A] [answered]
I noticed today, running an older version from mid-2020, that I'm not actually getting stereo from CS NG over the Pi's HDMI, only mono audio. I updated to NG Alpha7.2 2021-02-09 today, and I still have the same problem. Is there something that needs to be added to the config file or the CS ini file to force stereo? It's on a RBP 3B.

Maybe this has a simple solution, but I haven't found it, and I haven't noticed anyone else mentioning this issue. Any help would be greatly appreciated!

**Comment by Aaron407s** [comment, marked as answer]
After reflashing and copying over my config and crankshaft.ini file, the problem went away. Strange that updating over USB didn't fix it, but it's fixed nonetheless. Feel free to delete this question.

**Reply by matt2005** [reply]
How did you update before the reflash?

**Reply by Aaron407s** [reply]
I updated via USB drive. I had thought that the update process would effectively replace all files on the linux partition and most on the boot partition as well, save for a few retained files. At any rate, maybe this was just a 'one off' issue as it's working great now.
